Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -59,69 +59,49 @@ void addPoint(float ptX, float ptY) {
 
 __CGPath::~__CGPath() {
     if (_components) {
+		for (unsigned i = 0; i < _count; i++) {
```

Comment:
nit: please use spaces instead of tabs to match the rest of the codebase. To that end, we have setup clang-format for the project and ask that all PRs have it run prior to submission so that we can avoid silly formatting comments like this. :-)